File: java/src/test/java/lol/hub/codewars/valid_parentheses/SolutionTest.java
package lol.hub.codewars.valid_parentheses;

import org.junit.Test;

import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;

public class SolutionTest {
    @Test
    public void test1() {
        assertTrue(Solution.validParentheses("()"));
    }

    @Test
    public void test2() {
        assertFalse(Solution.validParentheses("())"));
    }

    @Test
    public void test3() {
        assertTrue(Solution.validParentheses("32423(sgsdg)"));
    }

    @Test
    public void test4() {
        assertFalse(Solution.validParentheses("(dsgdsg))2432"));
    }

    @Test
    public void test5() {
        assertTrue(Solution.validParentheses("adasdasfa"));
    }
}

File: go/twice_as_old/solution_test.go
package twice_as_old

import "testing"

func assert(t *testing.T, expected int, actual int) {
	if actual != expected {
		t.Errorf("actual: %v expected: %v", actual, expected)
	}
}

func Test1(t *testing.T) {
	assert(t, 22, TwiceAsOld(36, 7))
}

func Test2(t *testing.T) {
	assert(t, 5, TwiceAsOld(55, 30))
}

func Test3(t *testing.T) {
	assert(t, 0, TwiceAsOld(42, 21))
}

func Test4(t *testing.T) {
	assert(t, 20, TwiceAsOld(22, 1))
}

func Test5(t *testing.T) {
	assert(t, 29, TwiceAsOld(29, 0))
}
